examples/websocket-server/.env.example

```diff
@@ -0,0 +1,8 @@
+# OpenAI API Key
```

Comment:
File should be named `env.example` (not dot prefix) like other examples.

Reply:
change made